src/herder/TxSetFrame.cpp

```diff
@@ -96,38 +98,44 @@ SeqSorter(TransactionFramePtr const& tx1, TransactionFramePtr const& tx2)
 std::vector<TransactionFramePtr>
 TxSetFrame::sortForApply()
 {
-    vector<TransactionFramePtr> retList;
+    auto txQueues = buildAccountTxQueues();
 
-    vector<vector<TransactionFramePtr>> txBatches(4);
-    map<AccountID, size_t> accountTxCountMap;
-    retList = mTransactions;
-    // sort all the txs by seqnum
-    std::sort(retList.begin(), retList.end(), SeqSorter);
+    // build txBatches
+    // txBatches i-th element contains each i-th transaction for accounts with a
+    // transaction in the transaction set
+    std::list<std::deque<TransactionFramePtr>> txBatches;
```

Comment:
my compiler complains about not having list member in std
possibly missing include?

Reply:
fixed